101-recovery-services-backup-file-share/azuredeploy.json

```diff
@@ -0,0 +1,256 @@
+      "type": "string",
+      "defaultValue": "DailyBackupPolicy1",
+      "metadata": {
+        "description": "Name of the Backup Policy."
+      }
+    },
+    "scheduleRunTimes": {
+      "type": "array",
+      "defaultValue": [ "2020-11-04T05:30:00Z" ],
+      "metadata": {
+        "description": "Times in day when backup should be triggered. e.g. 01:00 or 13:00. Must be an array; however for File Shares only one value is valid. This will be used in LTR too for daily, weekly, monthly and yearly backup. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "timeZone": {
+      "type": "string",
+      "defaultValue": "UTC",
+      "metadata": {
+        "description": "Any valid timezone, for example: UTC, Pacific Standard Time. Refer: https://msdn.microsoft.com/en-us/library/gg154758.aspx (Ignore if using existing Backup Policy)."
+      }
+    },
+    "dailyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 30,
+      "metadata": {
+        "description": "Number of days for which the daily backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "daysOfTheWeek": {
+      "type": "array",
+      "defaultValue": [ "Sunday", "Tuesday", "Thursday" ],
+      "metadata": {
+        "description": "Array of days on which backup is to be performed for Weekly Retention. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "weeklyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 12,
+      "metadata": {
+        "description": "Number of weeks for which the weekly backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "monthlyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 60,
+      "metadata": {
+        "description": "Number of months for which the monthly backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "monthsOfYear": {
+      "type": "array",
+      "defaultValue": [ "January", "May", "September" ],
+      "metadata": {
+        "description": "Array of months on which backup is to be performed for Yearly Retention. (Ignore if using existing Backup Policy)."
+      }
+    },
+    "yearlyRetentionDurationCount": {
+      "type": "int",
+      "defaultValue": 10,
+      "metadata": {
+        "description": "Number of years for which the yearly backup is to be retained. (Ignore if using existing Backup Policy)."
+      }
+    }
+  },
+  "variables": {
+    "backupFabric": "Azure",
+    "backupManagementType": "AzureStorage",
+    "containerType": "StorageContainer",
+    "protectedItemType": "AzureFileShareProtectedItem",
+    "skuName": "RS0",
+    "skuTier": "Standard",
+    "workLoadType": "AzureFileShare"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.RecoveryServices/vaults",
+      "apiVersion": "2016-06-01",
+      "name": "[parameters('vaultName')]",
+      "location": "[parameters('location')]",
+      "sku": {
```

Comment:
move the sku settings to parameters since they will affect cost

Reply:
The sku settings do not affect the cost in case of this deployment.

Reply:
What does it affect?

Reply:
Didn't see a response here - if the sku of a resource does not affect the cost, what is it used for?

Reply:
The sku for a Recovery Services vault is a no-op and is present only because the guidelines mandate the field to be present for the resource creation. In the [portal view](https://ms.portal.azure.com/#create/Microsoft.RecoveryServices), a new Recovery Services vault creation doesn't ask the user for any costing-related parameters, and the sku parameters are automatically set to the ones I have used in the background.

Reply:
> The sku for a Recovery Services vault is a no-op and is present only because the guidelines mandate the field to be present for the resource creation. In the [portal view](https://ms.portal.azure.com/#create/Microsoft.RecoveryServices), a new Recovery Services vault creation doesn't ask the user for any costing-related parameters, and the sku parameters are automatically set to the ones I have used in the background.

Hey @bmoore-msft, I've responded to the conversation here, could you please review it?